lib/avo.rb

```diff
@@ -39,7 +39,7 @@ def initialize(resource_name)
     private
 
     def missing_resource_message(resource_name)
-      name = resource_name.to_s.downcase
+      name = resource_name.to_s.underscore.downcase
```

Comment:
```suggestion
      name = resource_name.to_s.underscore
```

We can remove the [`downcase`](https://apidock.com/ruby/String/downcase) since [`underscore`](https://apidock.com/rails/String/underscore) already lowercase the string

Reply:
Ok